serverless/aws/features/stepfunctions.py

```diff
@@ -237,11 +237,10 @@ def to_yaml(cls, dumper, data):
 class Parallel(Stage):
     yaml_tag = "!Parallel"
 
```

Comment:
Removing `End` since Parallel does not have `End` property.
https://www.serverless.com/plugins/serverless-step-functions#parallel

```
stepFunctions:
  stateMachines:
    yourParallelMachine:
      definition:
        Comment: "An example of the Amazon States Language using a parallel state to execute two branches at the same time."
        StartAt: Parallel
        States:
          Parallel:
            Type: Parallel
            Next: Final State
            Branches:
            - StartAt: Wait 20s
              States:
                Wait 20s:
                  Type: Wait
                  Seconds: 20
                  End: true
            - StartAt: Pass
              States:
                Pass:
                  Type: Pass
                  Next: Wait 10s
                Wait 10s:
                  Type: Wait
                  Seconds: 10
                  End: true
          Final State:
            Type: Pass
            End: true
```

Reply:
As far as I can see in AWS documentation it does have `End` https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-parallel-state.html

Reply:
Here is what I am seeing.

Without the above changes, 
- genereated serverless.yml configuration looks as below with `End` attribute.
- with the `End` attribute, when I deploy with ... sls deploy ... , I get the following error.
- with the changes, serverless.yml generated can deploy successfully.

```

✖ Stack user-data-export-service-dev failed to deploy (20s)
Environment: darwin, node 19.6.0, framework 3.30.1 (local) 3.21.0v (global), plugin 6.2.3, SDK 4.3.2
Credentials: Local, environment variables
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
✕ State machine "user-data-export" definition is invalid:
SCHEMA_VALIDATION_FAILED: /States/ExportUserDataFromAllServices/End is invalid. must be equal to one of the allowed values
```

```
....
...
definition:
        StartAt: ExportUserDataFromAllServices
        Comment: export user data
        States:
          ExportUserDataFromAllServices:
            Type: Parallel
            Branches:
            - StartAt: Userservice
              States:
                Userservice:
                  Type: Task
                  ....
            - StartAt: VideoConverterService
              States:
                VideoConverterService:
                  Type: Task
                  ...
            End: false
            ResultPath: null
            Next: PostExportProcess
            Catch:
            - ErrorEquals:
              - States.ALL
              Next: StateMachineErrorFallback
          PostExportProcess:
            Type: Task
            Resource:
              Fn::GetAtt:
              - PostExportProcessLambdaFunction
              - Arn
            End: true
            Catch:
            - ErrorEquals:
              - States.ALL
              Next: StateMachineErrorFallback
          StateMachineErrorFallback:
            Type: Pass
            Result: Error in state machine
            End: true
   
      
````